README.md

```diff
@@ -280,6 +303,7 @@ storm::config_map:
     - class: "com.verisign.storm.metrics.GraphiteMetricsConsumer"
```

Comment:
Should we consider renaming `GraphiteMetricsConsumer` now that we support more than one destination backend?